tap_github/client.py

```diff
@@ -224,6 +224,15 @@ def verify_access_for_repo(self):
             # Verifying for Repo access
             self.verify_repo_access(url_for_repo, repo)
 
+    def extract_orgs_from_config(self):
```

Comment:
I guess you should do code on https://github.com/singer-io/tap-github/pull/166 because what if provided org is invalid?

Reply:
Updated.